.cicdtemplate/.codemagic/codemagic.yaml

```diff
@@ -97,3 +99,67 @@ workflows:
       - *unit_test_on_template_xml
     artifacts:
       - *artifacts_template_xml
+
+  build-and-deploy-template-compose-staging:
+    name: Build and deploy template-compose staging to Firebase App Distribution
+    <<: *configure_environment
+    environment:
+      groups:
+        - firebase_credentials
+        - build_configure
```

Comment:
Can the team @nimblehq/android-chapter access the current CodeMagic config?

Reply:
@lydiasama I can not access this app https://codemagic.io/app/64afb77b452d69dc4c26bbe3/settings, is there a specific account for it?

Reply:
@luongvo The way to set it is very hard. I'm not sure about the permission or if I can share with anyone. @sleepylee Could you please help us? 🙏🏻

Reply:
🤔 it requires a team to be able to share, and establishing a team requires $$$ - you knows the rest. Maybe you could switch to use bot account (because bot has the access to the android-template repos?). The account can be shared by default

Reply:
@lydiasama you can get it from dev email

Device: Chrome on macOS
Verification code: 304432

Reply:
Updated setting up by bot account. 🙌🏻  @luongvo @sleepylee